src/segmantic/seg/transforms.py

```diff
@@ -0,0 +1,119 @@
+import warnings
+from pathlib import Path
+from typing import Dict, Optional, Sequence, Union
+
+import torch
+import yaml
+from monai.config.type_definitions import KeysCollection, NdarrayOrTensor
+from monai.networks import one_hot
+from monai.transforms.post.array import Ensemble
+from monai.transforms.post.dictionary import Ensembled
+from monai.transforms.transform import Transform
+from monai.utils import TransformBackends
+
+
+class SelectBestEnsemble(Ensemble, Transform):
```

Comment:
I will gladly help writing a test for this, but maybe you can give it a try? I would have a look how [monai tests MeanEnsembled](https://github.com/Project-MONAI/MONAI/blob/f23077dafafa32117d1224807e0cf7d6cbbce995/tests/test_mean_ensembled.py)